### ISSUE TITLE
Better API Response Format

### ISSUE THREAD

**Issue by orafaelfragoso** [AltCast/altcast-ticker#4]
We need a more friendly response format, like so:

```
[
	{
		exchange: 'BITFINEX',
		ticks: [{}, {}, {}, {}]
	},
	{
		exchange: 'POLONIEX',
		ticks: [{}, {}, {}, {}]
	}
]
```

Always returning an array of exchanges.

**Comment by orafaelfragoso** [comment]
Being more accurate:

```
[
	{
		exchange: 'BITFINEX',
		ticks: [
			{
				name: 'BTC',
				pairs: [
					{
						pair: 'BTC/USD',
						last_price: 18867,
						volume: 50555.80250399,
						high: 19891,
						low: 18622,
						daily_percent: -0.0168
					}
				]
			}
		]
	}
]
```

**Comment by andreabadesso** [comment]
Solved on commit 2979b76f2fb1175110a0c58447cb754869fcca4c
